lua/ulx/modules/sh/freeze.lua

```diff
@@ -0,0 +1,28 @@
+CATEGORY_NAME = "CFC"
+
+local function freezeProps( callingPlayer, targetPlayers ) 
+    local entities = ents.FindByClass( "prop_*" )
+    local entCount = 0
+    local isTarget = {}
+    for _, ply in pairs( targetPlayers ) do
+        isTarget[ply] = true
+    end
+
+    for _, ent in pairs( entities ) do
+        if isTarget[ent:CPPIGetOwner()] then
+            local physicsObj = ent:GetPhysicsObject()
+            if IsValid( physicsObj ) then
+                physicsObj:EnableMotion( false )
+                physicsObj:Sleep()
+                entCount = entCount + 1
+            end
+        end
+    end
+
+    ulx.fancyLogAdmin( callingPlayer, "#A froze "..entCount.." props owned by #T", targetPlayers )
```

Comment:
I think this should be reworked to have a count for each player instead of total count. How would this even look?
```
Phatso froze 1308 props owned by [<Player>, <Player>, <Player>]
```
?

How would the table be formatted?

Reply:
the rope clean command does this if you want an example

Reply:
You might wanna try out how i did it.